types/blob/blob.go

```diff
@@ -0,0 +1,82 @@
+package blob
+
+import (
+	"encoding/json"
+
+	"cosmossdk.io/errors"
+	"github.com/celestiaorg/nmt/namespace"
+
+	appns "github.com/rollkit/celestia-openrpc/types/namespace"
+)
+
+var (
+	ErrZeroBlobSize = errors.Register("blob", 11124, "cannot use zero blob size")
+)
+
+type Commitment []byte
+
+type Proof struct {
+	Start uint32   `protobuf:"varint,1,opt,name=start,proto3" json:"start,omitempty"`
+	End   uint32   `protobuf:"varint,2,opt,name=end,proto3" json:"end,omitempty"`
+	Nodes [][]byte `protobuf:"bytes,3,rep,name=nodes,proto3" json:"nodes,omitempty"`
+}
+
+type Blob struct {
```

Comment:
These types were recently updated in node, so need to copy those changes here.

Reply:
All the above types can be copied, except for `Commitment` which has deeper dependencies, so it would be better to leave that for now and just update the rest of the types.